src/Adapter/Presenter/AbstractLazyArray.php

```diff
@@ -241,6 +263,12 @@ public function offsetGet($index)
                     $this->methodCacheResults[$index] = $this->{$methodName}();
                 }
                 $result = $this->methodCacheResults[$index];
+            } elseif ($this->arrayAccessList[$index]['type'] === 'callable') {
+                if (!isset($this->methodCacheResults[$index])) {
+                    $callable = $this->arrayAccessList[$index]['value'];
+                    $this->methodCacheResults[$index] = call_user_func($callable[0], $callable[1] ?: null);
```

Comment:
My bad, wrong notation :D This one is better
```suggestion
                    $this->methodCacheResults[$index] = call_user_func($callable[0], $callable[1] ?? null);
```